cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -65,6 +69,10 @@ func NewKubeadmCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 			    You can then repeat the second step on as many other machines as you like.
 
 		`),
+
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return kubeadmutil.MaybeChroot(rootfsPath)
```

Comment:
let's rename `MaybeChroot` to just `Chroot` in symbols and comments.
`Maybe` is not really needed because the function has error handling.